src/client/App.js

```diff
@@ -33,6 +35,16 @@ function App() {
           >
             <Profile />
           </AuthenticatedRoute>
+          <AuthenticatedRoute
+            exact
+            path="/channel"
+            isAuthenticated={isAuthenticated}
+          >
+            <Channels />
+          </AuthenticatedRoute>
```

Comment:
What does channels do? I would remove the channels part and add the context to the Channel

Reply:
In my perspective Channels is the container where you have the channels overview.

In the bottom nav you can switch  between chats (channels overview) and profile overview.
<img width="387" alt="channels_view" src="https://user-images.githubusercontent.com/54849930/93484853-b4fdbc80-f902-11ea-911d-a4182705b32d.png">

Reply:
What you call channels is what we call overview and Malek has implemented that, it is already in develop.